src/main/java/xbot/common/controls/sensors/mock_adapters/MockLaserCAN.java

```diff
@@ -29,10 +33,12 @@ public MockLaserCAN(
 
     public void setDistance(double distanceMeters) {
         this.distanceMeters = distanceMeters;
+        this.measurementTime = XTimer.getFPGATimestamp();
     }
 
     @Override
     public void updateInputs(LaserCANInputs inputs) {
         inputs.distance = edu.wpi.first.units.Units.Meters.of(distanceMeters);
+        inputs.measurementLatency = Seconds.of(XTimer.getFPGATimestamp() - measurementTime);
```

Comment:
it's badly named, but there's now a getFPGATimestampTime() that returns a Time unit